docs/trino-connector/development.md

```diff
@@ -50,7 +50,8 @@ then you can see the `gravitino-trino-connecor` source files and directories in
 
 ![trino-gravitino-structure](../assets/trino/add-link.jpg)
 
-5. Change the `pom.xml` file in the `trino-gravitino` module accordingly. This is an example content of the `pom.xml` file in the `trino-gravitino` module.
+
+5. Add `<module>plugin/trino-gravitino</module>` to `trino/pom.xml` and change the `pom.xml` file in the `trino-gravitino` module accordingly. This is an example content of the `pom.xml` file in the `trino-gravitino` module.
 
```

Comment:
We need to ensure the version of trino-root in this pom.xml file matches the version of Trino.

Reply:
Added.